operator/src/main/resources/application.properties

```diff
@@ -38,6 +38,14 @@ ingresscontroller.peak-throughput-percentage=40
 # old sizing - this should go higher, but we should need more memory
 ingresscontroller.max-ingress-connections=75000
 
+# Enables haproxy option contstats in the kas ingress so that stats are reported live rather than at connection close (RFE-3007 provide a proper mechanism to enable this option).
+ingresscontroller.ingress-container-command=/usr/bin/bash,-c,awk '{print $0} /^defaults$/ {print \"  option contstats\"}' < $TEMPLATE_FILE > /tmp/haproxy-config.template; exec /usr/bin/openshift-router --v=2 --template /tmp/haproxy-config.template
```

Comment:
🤠 

And I thought my suggestion of doing it as an image build was pushing the bounds of dirty hacks.  This is next level evil genius stuff.

Reply:
Is this safe in the face of a re-started pod? If the pod is re-started we are going to end up with duplicated `option contstats` entries aren't we?

(Firing up a openshift cluster now to test this)

Reply:
> 🤠
> 
> And I thought my suggestion of doing it as an image build was pushing the bounds of dirty hacks. This is next level evil genius stuff.

![image](https://user-images.githubusercontent.com/18440250/178684803-2219ece6-8474-492d-afe2-1683577fa6c3.png)

Reply:
Good point re: ` /tmp/haproxy-config.Template` I hadn't noticed the path change